docs/4.17.21.html

```diff
@@ -1699,12 +1699,12 @@ <h4>Arguments</h4>
ric">36</span><span class="delimiter">,</span>&#xA0;<span class="string">&apos;active&apos;</span>:&#xA0;true&#xA0;}<span class="delimiter">,</span></div><div>&#xA0;&#xA0;{&#xA0;<span class="string">&apos;user&apos;</span>:&#xA0;<span class="string">&apos;fred&apos;</span><span class="delimiter">,</span>&#xA0;&#xA0;&#xA0;<span class="string">&apos;age&apos;</span>:&#xA0;<span class="numeric">40</span><span class="delimiter">,</span>&#xA0;<span class="string">&apos;active&apos;</span>:&#xA0;false&#xA0;}</div><div>];</div><div>&#xA0;</div><div>_<span class="delimiter method">.</span><span class="name">filter</span>(users<span class="delimiter">,</span>&#xA0;<span class="type">function</span>(o)&#xA0;{&#xA0;return&#xA0;!o<span class="delimiter">.</span>active;&#xA0;});</div><div><span class="comment">//&#xA0;=&gt;&#xA0;objects&#xA0;for&#xA0;[&apos;fred&apos;]</span></div><div>&#xA0;</div><div><span class="comment">//&#xA0;The&#xA0;`_.matches`&#xA0;iteratee&#xA0;shorthand.</span></div><div>_<span class="delimiter method">.</span><span class="name">filter</span>(users<span class="delimiter">,</span>&#xA0;{&#xA0;<span class="string">&apos;age&apos;</span>:&#xA0;<span class="numeric">36</span><span class="delimiter">,</span>&#xA0;<span class="string">&apos;active&apos;</span>:&#xA0;true&#xA0;});</div><div><span class="comment">//&#xA0;=&gt;&#xA0;objects&#xA0;for&#xA0;[&apos;barney&apos;]</span></div><div>&#xA0;</div><div><span class="comment">//&#xA0;The&#xA0;`_.matchesProperty`&#xA0;iteratee&#xA0;shorthand.</span></div><div>_<span class="delimiter method">.</span><span class="name">filter</span>(users<span class="delimiter">,</span>&#xA0;[<span class="string">&apos;active&apos;</span><span class="delimiter">,</span>&#xA0;false]);</div><div><span class="comment">//&#xA0;=&gt;&#xA0;objects&#xA0;for&#xA0;[&apos;fred&apos;]</span></div><div>&#xA0;</div><div><span class="comment">//&#xA0;The&#xA0;`_.property`&#xA0;iteratee&#xA0;shorthand.</span></div><div>_<span class="delimiter method">.</span><span class="name">filter</span>(users<span class="delimiter">,</span>&#xA0;<span class="string">&apos;active&apos;</span>);</div><div><span class="comment">//&#xA0;=&gt;&#xA0;objects&#xA0;for&#xA0;[&apos;barney&apos;]</span></div><div>&#xA0;</div><div><span class="comment">//&#xA0;Combining&#xA0;several&#xA0;predicates&#xA0;using&#xA0;`_.overEvery`&#xA0;or&#xA0;`_.overSome`.</span></div><div>_<span class="delimiter method">.</span><span class="name">filter</span>(users<span class="delimiter">,</span>&#xA0;_<span class="delimiter method">.</span><span class="name">overSome</span>([{&#xA0;<span class="string">&apos;age&apos;</span>:&#xA0;<span class="numeric">36</span>&#xA0;}<span class="delimiter">,</span>&#xA0;[<span class="string">&apos;age&apos;</span><span class="delimiter">,</span>&#xA0;<span class="numeric">40</span>]]));</div><div><span class="comment">//&#xA0;=&gt;&#xA0;objects&#xA0;for&#xA0;[&apos;fred&apos;,&#xA0;&apos;barney&apos;]</span></div></pre></div>
```

Comment:
> leaving comment explaining this diff since its unreadble, these diffs are from the doc build output from lodash repo

Added a 5th example demonstrating how to combine multiple predicates with OR logic:
 ```js
 // Combining several predicates using `_.overEvery` or `_.overSome`.
  _.filter(users, _.overSome([{ 'age': 36 }, ['age', 40]]));
  // => objects for ['fred', 'barney']
```
https://github.com/lodash/lodash/blob/9648f692b0fc7c2f6a7a763d754377200126c2e8/lodash.js#L9235-L9238

Reply:
Relevant PR https://github.com/lodash/lodash/pull/4514